modules/@apostrophecms/modal/ui/apos/components/AposModal.vue

```diff
@@ -232,47 +250,26 @@ export default {
       this.$emit('esc');
     },
     trapFocus () {
-      // Adapted from https://uxdesign.cc/how-to-trap-focus-inside-modal-to-make-it-ada-compliant-6a50f9a70700
-      // All the elements inside modal which you want to make focusable.
-      const focusableElements = [
-        'button',
+      const elementSelectors = [
+        '[tabindex]',
```

Comment:
Right, and we'll put this on tiptap presumably (I know tiptap will have its own constellation of accessibility problems too and that's OK for another ticket)